content/en/environments/_index.html

```diff
@@ -0,0 +1,105 @@
+---
+title: "Fission Environments"
+linkTitle: "Environments"
+description: "Runtime for different languages"
+images: ["images/og-image-fission.png"]
+---
+
+<div class="container push-down mb-5">
+  <div class="row mb-2">
+    <div class="col-12">
+      <h1 class="font-weight-bold">Environments</h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <h5><label class="d-block" for="search">Search</label></h5>
+        <input type="search" placeholder="Search query" class="form-control" 
+          id="search" oninput="updateEnvItems()">
+      </div>
+    </div>
+    <div class="col">
+      <div class="form-group">
+        <h5><label class="d-block" for="filter-lang">Filter by language</label></h5>
+        <select class="form-control" id="filter-lang" onchange="updateEnvItems()">
+          <option value="" selected disabled>Loading...</option>
+        </select>
+      </div>
+    </div>
+  </div>
+
+  <div class="row" id="env-list"></div>
+</div>
+
+<script type="text/javascript">
+  const envs = {{< readfile file="static/data/environments.json" >}};
```

Comment:
This will work for file fetched over http ?
 Eg. [environments.fission.io](http://environments.fission.io/) etc

Reply:
No. `readfile` only supports local files. For fetching over http, we will have to use `fetch` with the corresponding endpoint.